api/src/main/java/org/apache/cloudstack/api/command/admin/autoscale/CreateAutoScaleCounterCmd.java

```diff
@@ -34,7 +34,7 @@
 
 @APICommand(name = "createCounter", description = "Adds metric counter for VM auto scaling", responseObject = CounterResponse.class,
```

Comment:
```suggestion
@APICommand(name = "createAutoScaleCounter", description = "Adds metric counter for VM auto scaling", responseObject = CounterResponse.class,
```

this needs to change for the API name to change @erikbocks .